kie-wb-tests/kie-wb-tests-rest/src/main/java/org/kie/wb/test/rest/client/RestWorkbenchClient.java

```diff
@@ -337,11 +313,18 @@ public RemoveRepositoryFromOrganizationalUnitRequest removeRepositoryFromOrganiz
                 .request().post(createEntity(""), requestType);
```

Comment:
This private method should be used, because path isn't exist. Can you delete it, please?